### PR TITLE
Added the screenshots and updated the readme screenshots

### LINKED ISSUE #116: Add up to date screenshots of the desktop/larger site
**Tasks:**
- [x] add some replacements to the pictures on `readme.md`
- [x] add as many screenshots as you can in `./docs/screenshots.md`

### PR DESCRIPTION
**Pull request by Shivashish-rwt**
## 🛠️ Fixes issue(s)

Closes #116 

## 👨‍💻 Change(s) proposed

Added the updated the screenshots in `README.md` and added the Screenshots in `./docs/SCREENSHOTS.md`

## ✔️ Checklist (Check all the applicable boxes) 

- [x ] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

**Comment by Shivashish-rwt** [comment]
@Ngoakor12 
Please review the PR.

**Review by Ngoakor12 — CHANGES_REQUESTED** [review]
Thanks you for open a PR and being patient, here are my thoughts...
- One big thing, The screenshots to the previous site should not be added, I think the links is enough and anything more than that seems like I'm taking a dig at it.
- The `screenshots.md` file is in the wrong place, create a folder on the root named `docs` and create a `screenshot.md` file inside, you may also add the `.png` files in a `screenshots` folder inside `docs` folder
- In the `readme.md` in in the root, add a screenshot of the resources page and bookmarks page under the `screenshots` heading, and also link the the screenshots.md file with something like..."More screenshots" text
- Screenshots folder name should be all lowercase to keep everything consistent.

**Comment by Shivashish-rwt** [comment]
@Ngoakor12 
I have made the changes as u told. Please let me know if more changes required.

**Review by Ngoakor12 — CHANGES_REQUESTED** [review]
- Move the `screenshots.md` from `./backend/src/docs` to `./docs`
- There still some screenshots pngs from the original ACN website
- `./screenshots` folder should be in the `./docs` folder
- The current two pictures in `./readme.md` should be replaced with screenshots of how the site currently looks

**Comment by Shivashish-rwt** [comment]
@Ngoakor12 
Please review.

**Review by Ngoakor12 — CHANGES_REQUESTED** [review]
Almost there, just one tiny thing you've missed twice...
You should not touch the backend folder. **The `SCREENSHOTS.md` file that's in the `./backend/src/docs` folder should be move to the `./docs` folder you created in the root of the project**

**Comment by Shivashish-rwt** [comment]
@Ngoakor12 
Moved the `SCREENSHOTS.md` to `./docs`. Please check.

File: README.md [hunk 1]
```diff
@@ -38,8 +38,11 @@ To solve these problems that only someone who visits the site frequently notices
 - Server-side and client-side pagination.
 
 ## Screenshots
-![www ngoako com_coding-resource-finder_(myDevice highres desktop)](https://user-images.githubusercontent.com/54069197/155377612-66abb465-783e-46c7-be4c-b98b1f74848f.png)
-![www ngoako com_coding-resource-finder_(myDevice highres desktop) (1)](https://user-images.githubusercontent.com/54069197/155377548-f7db8a17-ee1c-4aa0-b331-e1d6214e78b1.png)
+
+![www ngoako com_coding-resource-finder_(myDevice highres desktop)](./docs/screenshots/Screenshot%20(141).png)
+![www ngoako com_coding-resource-finder_(myDevice highres desktop) (1)](./docs/screenshots/Screenshot%20(156).png)
+
+[More screenshots](./docs/screenshots/)
```
Comment (Ngoakor12) [inline README.md]: please link to the `SCREENSHOTS.md` instead of the screenshots images folder for the "More screenshot" link

**Review by Ngoakor12 — CHANGES_REQUESTED** [review]

**Comment by Shivashish-rwt** [comment]
@Ngoakor12 
Linked `SCREENSHOTS.md`.

**Comment by Ngoakor12** [comment]
Much better, thanks for the contribution @Shivashish-rwt

**Review by Ngoakor12 — APPROVED** [review]

**Comment by Shivashish-rwt** [comment]
@Ngoakor12 
Thanks for giving me opportunity to contribute.
This is my first time contributing to open source.  Could u give me some advice or anything i need to learn in order to become better??

**Comment by Ngoakor12** [comment]
> @Ngoakor12 Thanks for giving me opportunity to contribute. This is my first time contributing to open source. Could u give me some advice or anything i need to learn in order to become better??

- Don't work on the master branch of the branch you forked, rather create a separate branch
- And take time to understand the issue and read feedback carefully, because you did miss a lot of stuff, but what matters is you got there in the end :+1:

**Comment by Shivashish-rwt** [comment]
Thanks for advice.

**Pull request merged**